### ISSUE TITLE
[BUG -> iOS] The top bar in chats can be transparent 

### ISSUE THREAD

**Issue by socksprox** [simplex-chat/simplex-chat#810]
When entering a chat the top bar is invisible for a short time. This is only the case if the chat has some messages i experienced. Here a video

**Comment by PalmTamino** [comment]
Video: https://streamable.com/w0jes0

**Comment by epoberezkin** [comment]
yep, swiftui is far from perfect... we won't be fixing it for now...

**Comment by PalmTamino** [comment]
> yep, swiftui is far from perfect... we won't be fixing it for now...

I don’t have this issue anymore tho😅 seems to be fixed by itself after a year….😆

**Comment by kimg45** [comment]
Seems to be fixed on iOS 17.4 from what I can tell. Haven't seen this bug at all from using the app for a while now and couldn't get it to appear despite trying really hard. Anyone else able to confirm?

**Comment by kimg45** [comment]
The top bar seems to light up a bit when scrolling and go back to full black at the bottom of the chat in dark mode. It darkens a bit when scrolling and settles back to full white at the bottom of the chat in light mode. Not transparent but very strange.

**Comment by friadev** [comment]
I guess I'm not really seeing this issue anymore on iOS 18, think it might be fixed?

**Comment by socksprox** [comment]
Same. Closing
